client/index.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// eslint-disable-next-line import/extensions
+import App from './App.jsx';
+
+ReactDOM.render(<App />, document.querySelector('#about'));
```

Comment:
Maybe you were having one of the same problems that I was having, which was that I had the App component in my index.js file and apparently the tests can't reference the file that calls ReactDOM.render. Anyway, all looks good!